cdap-app-fabric/src/main/java/io/cdap/cdap/support/status/SupportBundleStatus.java

```diff
@@ -25,37 +25,171 @@
  * Status when generating Support bundle.
  */
 public class SupportBundleStatus {
-  /** UUID of the bundle status object describes */
+  /**
+   * UUID of the bundle status object describes
+   */
   private final String bundleId;
-  /** status of bundle collection (IN_PROGRESS/FINISHED/FAILED) */
+  /**
+   * status of bundle collection (IN_PROGRESS/FINISHED/FAILED)
+   */
   private final CollectionState status;
-  /** Failed bundle describes the failure */
+  /**
+   * Failed bundle describes the failure
+   */
   private String statusDetails;
-  /** when bundle collection was started */
+  /**
+   * when bundle collection was started
+   */
   private final long startTimestamp;
-  /** FINISHED/FAILED bundles when bundle collection was completed */
+  /**
+   * FINISHED/FAILED bundles when bundle collection was completed
+   */
   private long finishTimestamp;
```

Comment:
We should probably do null to indicate it's not finished. If we do so, do same for start